upflux-client/src/App.tsx

```diff
@@ -28,10 +30,25 @@ export const App = () => {
                 </section>
                 <Footer />
               </>
-            } 
+            }
           />
 
+          {/* Login route */}
           <Route path="/login" element={<LoginComponent />} />
+
```

Comment:
for the next PR or if you like. you can do here itself. Create File called LoginRoute.tsx and just in your return put the "Routing".. and use those components and fill up your app.tsx and we go forward we would be having a lot of routes and we do not want to clog up app.tsx file.

My suggestion is you should do it now.